src/test/mock-request.js

```diff
@@ -1,15 +1,15 @@
 const mockReq = {
-  body: {
-    requestor: {
-      role: 'Administrator',
-      permissions: {
-        frontPermissions: ['addDeleteEditOwners'],
-        backPermissions: [],
-      },
-      requestorId: '65cf6c3706d8ac105827bb2e', // this one matches the id of the db/createUser for testing purposes
+    body: {
+        requestor: {
+            role: 'Administrator',
+            permissions: {
+                frontPermissions: ['addDeleteEditOwners', 'postTeam'],
+                backPermissions: ['postTeam'],
+            },
+            requestorId: '65cf6c3706d8ac105827bb2e', // this one matches the id of the db/createUser for testing purposes
+        },
     },
-  },
-  params: { userid: '5a7e21f00317bc1538def4b7', userId: '5a7e21f00317bc1538def4b7' },
+    params: { userid: '5a7e21f00317bc1538def4b7', userId: '5a7e21f00317bc1538def4b7', teamId: '5a8e21f00317bc' },
```

Comment:
Please, run all unit tests when you update these kind of files that are shared between many test suites